### PR TITLE
[amp-story] manual test for dfp

### PR DESCRIPTION
**Pull request by calebcordry**
Example story using double-click config to ensure everything is going well 😁

### CODE REVIEW

**Comment by calebcordry** [comment]
NOTE: we are scrubbing height, width, layout here:
https://github.com/ampproject/amphtml/blob/54e69d1654da819dd15fbd3127f655679ab0fc2f/extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js#L296-L301

We can decide what we should do permanently for DFP but until then you may want to just comment that part out to manually test

File: test/manual/amp-story-doubleclick.amp.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    <amp-story standalone bookend-config-src="../../examples/amp-story/bookend.json">
+        <amp-story-auto-ads>
+            <script type="application/json">
+              {
+                "ad-attributes": {
+                  "type": "doubleclick",
+                  "width": "320",
+                  "height": "50",
+                  "layout": "fixed",
+                  "data-slot": "/30497360/samfrank_native_v2_a4a"
```
Comment (calebcordry) [inline test/manual/amp-story-doubleclick.amp.html]: Happy to change any of this if you have an example you would rather use.

**Comment by calebcordry** [comment]
cc/ @keithwrightbos @jeffkaufman

**Comment by calebcordry** [comment]
Talked to @lannka offline. We only want to support `layout=fill` so we are keeping the current behavior and auto-inserting that. We will continue to destroy any user defined height, width or layout attributes.

**Comment by glevitzky** [comment]
I copied the files here to my branch in PR #14770, and I noticed that the `amp-story` attribute is not being added to the ad slot element. Has the code for that not been written yet, or is this a bug?

**Comment by calebcordry** [comment]
@glevitzky Did you try rebasing? It has only been added in the last couple days (maybe before you started this PR)

**Pull request merged**